go/cmd/dolt/commands/cherry-pick.go

```diff
@@ -97,12 +97,12 @@ func (cmd CherryPickCmd) Exec(ctx context.Context, commandStr string, args []str
 		return HandleVErrAndExitCode(verr, usage)
 	}
 
-	verr := cherryPick(ctx, dEnv, cherryStr)
+	verr := cherryPick(ctx, dEnv, cliCtx, cherryStr)
```

Comment:
How is cherryPick needed for this change?

Reply:
CherryPick calls Add internally. If we migrate Add to use `cliCtx` but we don't migrate CherryPick to pass in `cliCtx`, then CherryPick will break.